guide/portuguese/ruby/index.md

```diff
@@ -86,7 +86,7 @@ Da linha de comando:
 Ruby está bem [documentado](https://www.ruby-lang.org/en/documentation/) . Esses documentos incluem tutoriais, guias, referências e informações meta para o idioma.  
 Outro recurso importante para documentação é o [Ruby Doc](http://ruby-doc.org/core-2.3.0/) . Você deve visitar este [link](https://github.com/airbnb/ruby) para saber mais sobre o guia de estilo Ruby, escrito por desenvolvedores do AirBnB.
 
-Uma leitura recomendada para iniciantes em Ruby é [o guia do Why's (Poignant) para Ruby](https://poignant.guide/) Este livro é incomum entre os livros de programação. Com um monte de humor estranho e faixas laterais narrativas que às vezes são completamente alheios ao tema, este consegue manter os leitores entretidos enquanto aprendem o básico de Ruby.
+Uma leitura recomendada para iniciantes em Ruby é [o guia do Why's (Poignant) para Ruby](https://poignant.guide/). Este livro é incomum entre os livros de programação. Com um monte de humor estranho e faixas laterais narrativas que às vezes são completamente alheios ao tema, este consegue manter os leitores entretidos enquanto aprendem o básico de Ruby. É importante notar que o Poignant Guide foi publicado há alguns anos, e pode estar defasado. Mas mesmo assim a leitura é divertida e ah, tem [tradução para o português](http://why.carlosbrando.com/).
```

Comment:
> Mas mesmo assim a leitura é divertida e ah, tem

Corrigir este tipo de escrita

Reply:
@fabiotp91 Do you have a suggestion of what this should be reworded to?  If so, then you can click make a suggestion by clicking on the line with the issue and then adding a comment with the actually suggestion to use.
You would type:
````suggestion
   your suggestion goes here
````
If it is a small adjustment needed and you have write access, then you can make the change yourself on behalf of the user.  I would only recommend doing this for small changes though.

Reply:
```suggestion
Uma leitura recomendada para iniciantes em Ruby é [o guia do Why's (Poignant) para Ruby](https://poignant.guide/). Este livro é incomum entre os livros de programação. Com um monte de humor estranho e faixas laterais narrativas que às vezes são completamente alheios ao tema, este consegue manter os leitores entretidos enquanto aprendem o básico de Ruby. É importante notar que o Poignant Guide foi publicado há alguns anos, e pode estar defasado. Mas se preferir tem a [tradução para o português](http://why.carlosbrando.com/).
```

Reply:
@fabiotp91 If line 89 is the only thing that needs to be changed, I will change it.  FYI - I will be removing `Mas se preferir tem a [tradução para o português](http://why.carlosbrando.com/).`, because the link is not `https` and we prefer links to content to be secure.  

If I make the suggested change on line 89, is everything else fine?  I will resolve the merge conflicts.

Reply:
Yes, if you remove that complete sentence there's no problem, the sentence only says that you can read the comic in Portuguese if you prefer.